faasmcli/faasmcli/tasks/libs.py

```diff
@@ -66,15 +62,9 @@ def native(ctx, clean=False):
     build_cmd_str = " ".join(build_cmd)
     print(build_cmd_str)
 
-    res = call(build_cmd_str, shell=True, cwd=build_dir)
-    if res != 0:
-        raise RuntimeError("Failed to build native tools")
-
-    res = call("ninja", shell=True, cwd=build_dir)
-    if res != 0:
-        raise RuntimeError("Failed to make native tools")
-
-    call("sudo ninja install", shell=True, cwd=build_dir)
+    run(build_cmd_str, shell=True, cwd=build_dir, check=True)
+    run("ninja", shell=True, cwd=build_dir, check=True)
+    run("sudo ninja install", shell=True, cwd=build_dir, check=True)
```

Comment:
Changed all calls to `call` and `check_output` to the more modern (and concise) `run`. Considered a separate PR but not sure it's worth it